clj/test/slipstream/ui/util/pattern_test.clj

```diff
@@ -22,7 +22,7 @@
 
 (defn- match-not-forbidden
   [matches? s]
-  (let [expected  (if matches? s nil)
+  (let [expected  (when matches? s)
```

Comment:
All of `(if matches? s nil)`, `(if matches? s)` and `(when matches? s)` do the same thing in the end.  The reason for using `when` rather than `if` is that `when` takes only a single form, so it is semantically clearer (subjectively) that only a single form was intended.